doc/whats_new/v1.2.rst

```diff
@@ -42,7 +42,13 @@ random sampling procedures.
   :func:`linear_model._sgd_fast._plain_sgd` which is used by :class:`linear_model.SGDRegressor`
   and :class:`linear_model.SGDClassifier`. The old condition did not disambiguate between
   training and validation set and had an effect of overscaling the error tolerance.
-  This has been fixed in :pr:`23798` by :user:`Harsh Agrawal <Harsh14901>`
+  This has been fixed in :pr:`23798` by :user:`Harsh Agrawal <Harsh14901>`.
+
+- |Fix| :class:`model_selection.GridSearchCV` and
+  :class:`model_selection.RandomizedSearchCV` will rank as wort equally
+  performers the fit/scoring routines that fail and return `np.nan` scores.
+  Previous failing routines were ranked arbitrarily differently without ties.
+  :pr:`24543` by :user:`Guillaume Lemaitre <glemaitre>`.
```

Comment:
What do you think of this:

```suggestion
- |Fix| :class:`model_selection.GridSearchCV` and
  :class:`model_selection.RandomizedSearchCV` will rank fit/scoring routines
  that raise an exception as equal worst. Previously they were ranked as worst performers
  in an arbitrary order.
  :pr:`24543` by :user:`Guillaume Lemaitre <glemaitre>`.
```

I think the key thing to get across to people is that failing runs were always ranked as the worst results and will continue to be ranked worst. The change is that they will now be tied for last place, instead of there being an arbitrary order to them. It is tricky to pack all that into a short paragraph :-/